act/accordions.md

```diff
@@ -0,0 +1,93 @@
+# Accordions
+
+## Rule Identifier
+
+gel-foundations-accordions
+
+## Rule Description
+
+The accordion consists of a number of collapsed sections, each with a button to expand that section's content.
+
+This rule tests that when an accordion is included in an HTML page it meets the requirement of the BBC GEL Accordion component.
+
+## Rule Type
+
+atomic
+
+## Accessibility Requirements Mapping
+
+- [BBC GEL Accordions](https://bbc.github.io/gel/components/accordions/)
+  - Required for conformance to BBC GEL
+  - Outcome mapping:
+    - Any failed outcomes: success criterion is not satisfied
+    - All passed outcomes: success criterion needs further testing
+    - An inapplicable outcome: success criterion needs further testing
+
+## Input Aspects
+
+DOM Tree
+CSS Styling
+
+## Applicability
+
+This rule applies to any HTML page that includes a BBC GEL Accordion component.
+
+## Expectations
+
+@NOTE: This assumes we are documenting a valid Accordion, not specifically the [Accordion reference implementation](https://bbc.github.io/gel/components/accordions/#reference-implementation)
+
+### Expectation 1
+
+Each accordion component consist of two or more accordion items; an accordion item is a combination of content that can be expanded and collapsed and the control that allows the content to be expanded and collapsed.
+
+### Expectation 2
+
+Each accodion item has a keyboard, touch, and pointing device accessible control that is used to expand and collapse the accordion item.
+
+### Expectation 3
+
+Accordion item controls are visible in both the expanded and collapsed states of the accordion item.
+
+### Expectation 4
+
+Accrodion item controls have a clear visual style that marks them out as interactive and related to the content that they control when the content is in the expanded state.
+
+### Expectation 5
+
+Each accordion item control has a visual indication of the current expanded / collapsed state of the accodion item.
+
+### Expectation 6
+
+Each accordion item control has content, preferably text, that describes the content that it controls.
+
```

Comment:
Any guidance on cognitive accessibility for control content: Must be conceptually clear and distinct from other control "labels" (are they _labels_?) for example?